### ISSUE TITLE
RiotException: Premature end of file

### ISSUE THREAD

**Issue by carueda** [mmisw/orr-portal#99]
The following was recently registered by @lewismc at the COR instance:
   http://cor.esipfed.org/ont/~lmcgibbn/morning1

I just tried it and got:

```json
{
  "uri": "http://cor.esipfed.org/ont/~lmcgibbn/morning1",
  "version": "20170412T000427",
  "format": "rj",
  "error": "Cannot create requested ontology format: org.apache.jena.riot.RiotException: Premature end of file."
}
```

![image](https://cloud.githubusercontent.com/assets/556505/24937622/db414e9c-1ee6-11e7-81ba-0e01c1d8e297.png)

**Comment by lewismc** [comment]
By removing the parentheses of the underlying CSV data from which I was generating Turtle, i was successfully able to upload. 
http://cor.esipfed.org/ont/~lmcgibbn/morning1v2
I think this issue can be closed alongside #98, we can just ensure that parentheses are not present within the source data.

**Comment by lewismc** [comment]
@carueda can you please delete the [offending record](http://cor.esipfed.org/ont/~lmcgibbn/morning1)? Thanks

**Comment by carueda** [comment]
> @carueda can you please delete the offending record? Thanks

Done (`http://cor.esipfed.org/ont/~lmcgibbn/morning1` unregistered)

> I think this issue can be closed alongside #98, we can just ensure that parentheses are not present within the source data.

Sounds good. Just closed this and that.
